docs/running-headscale-container.md

```diff
@@ -54,7 +62,7 @@ docker run \
   --detach \
   --rm \
   --volume $(pwd)/config:/etc/headscale/ \
-  --publish 127.0.0.1:8080:8080 \
+  --publish 0.0.0.0:8080:8080 \
```

Comment:
This was intentionally left on 127.0.0.1. So people won't publish this service by default and have to think before the do. So rather on the approach keep it save until you really want to publish it.

Reply:
Fixed in [3272feb](https://github.com/juanfont/headscale/pull/521/commits/3272febfb3e42081d14b620b0863f3c6defa7440)